libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.scss

```diff
@@ -1,64 +1,238 @@
+  --sky-override-text-editor-top-toolbar-bottom-padding: 0;
+  --sky-override-text-editor-wrapper-background-color-disabled: var(
+    --sky-background-color-disabled
+  );
+  --sky-override-text-editor-wrapper-padding: 1rem;
+}
+
+@include compatMixins.sky-default-overrides('.sky-text-editor-hint-text') {
+  --sky-override-text-editor-hint-text-space: 5px;
+}
+
+@include compatMixins.sky-modern-overrides('.sky-text-editor', false) {
+  --sky-override-text-editor-background-color: var(--modern-color-white);
+  --sky-override-text-editor-border-color: #{$sky-theme-modern-border-color-neutral-medium};
+  --sky-override-text-editor-focus-shadow: #{$sky-theme-modern-elevation-1-shadow-size
+    $sky-theme-modern-elevation-1-shadow-color};
+  --sky-override-text-editor-label-padding: 10px 10px 0 10px;
+  --sky-override-text-editor-toolbar-section-padding: #{$sky-theme-modern-space-sm};
+  --sky-override-text-editor-top-toolbar-bottom-padding: 0;
+}
+
+sky-text-editor.sky-form-field-stacked {
   display: block;
 }
 
 .sky-text-editor {
+  border-radius: var(
+    --sky-override-text-editor-border-radius,
+    var(--sky-border-radius-s)
+  );
+
+  box-shadow: var(
+    --sky-override-text-editor-box-shadow,
+    inset 0 0 0 var(--sky-border-width-input-base)
+      var(
+        --sky-override-text-editor-border-color,
+        var(--sky-color-border-input-base)
+      )
+  );
+
   &.sky-text-editor-disabled {
-    .sky-toolbar *,
-    .sky-text-editor-wrapper * {
-      color: var(--sky-text-color-deemphasized) !important;
-    }
+    background-color: var(--sky-color-background-input-disabled);
+    cursor: var(--sky-override-text-editor-disabled-cursor, not-allowed);
   }
 
-  .sky-text-editor-label-wrapper {
-    display: flex;
+  .sky-text-editor-label-wrapper:not(:empty) {
+    display: var(--sky-override-text-editor-label-display, inline-block);
+    padding: var(
+      --sky-override-text-editor-label-padding,
+      var(--sky-space-inset-input_label-top-m)
+        var(--sky-space-inset-input-right-m) 0
+        var(--sky-space-inset-input-left-m)
+    );
     width: 100%;
+    color: var(
+      --sky-override-text-editor-label-color,
+      var(--sky-color-text-deemphasized)
+    );
+
+    label {
+      font-size: var(
+        --sky-override-text-editor-label-font-size,
+        var(--sky-font-size-input-label)
+      );
+      font-style: var(--sky-font-style-input-label);
+      font-weight: var(
+        --sky-override-text-editor-label-font-weight,
+        var(--sky-font-weight-input-label)
+      );
+      letter-spacing: var(--sky-font-letter_spacing-input-label);
+      line-height: var(
+        --sky-override-text-editor-label-line-height,
+        var(--sky-font-line_height-input-label)
+      );
+
+      margin-bottom: var(--sky-override-text-editor-label-margin-bottom, 0);
+    }
   }
 
   .sky-text-editor-wrapper {
     display: flex;
     flex-wrap: wrap;
-    background-color: white;
+    background-color: var(
```

Comment:
The background color is set here - but I don't see it set on the toolbar or label portion.